frontend/dockerfile/docs/reference.md

```diff
@@ -1434,9 +1434,8 @@ ADD arr[[]0].txt /dest/
 
 When using a local tar archive as the source for `ADD`, and the archive is in a
 recognized compression format (`gzip`, `bzip2` or `xz`, or uncompressed), the
-archive is decompressed and extracted into the specified destination. Only
-local tar archives are extracted. If the tar archive is a remote URL, the
-archive is not extracted, but downloaded and placed at the destination.
+archive is decompressed and extracted into the specified destination. Local tar
```

Comment:
Just above this line there is a related `(...)` about compression format support, but worded differently, may be better to be consistent?

This one mentions `xz` support, but the format support is also implied as only the 3 codecs or uncompressed, as opposed to the newer wording where format support is ambiguous.